requirements/specifications/lifecycle/presentation.md

```diff
@@ -0,0 +1,121 @@
+| `offscreen`  | The app is has it's graphics surface attached, but not displayed on the screen at the moment, e.g. scrolled out of view |
+| `scaled`     | The app is displayed at a size smaller than the entire screen but at least 25% of the width or height |
+| `thumbnail` | The app is displayed at a size smaller than 25% of the width or height of the entire screen |
+| `loading` | The platform is displaying a loading screen while the app prepares to be activated | 
+| `none`    | The app does not have it's graphics surface attached to the screen |
+
+### 3.1. Display vs Lifecycle
+Each Lifecycle state only supports certain display states:
+
+| Lifecycle      | Supported Displays                               |
+|----------------|--------------------------------------------------|
+| `initializing` | `none`, `loading`                                |
+| `active`       | `fullscreen`, `scaled`, `thumbnail`, `offscreen` |
+| `running`      | `none`, `loading`                                |
+| `suspended`    | `none`, `loading`                                |
+| `sleeping`     | `none`, `loading`                                |
+
+See [Picture-in-picture](#6-picture-in-picture-video) and [Background Audio](#5-background-audio) for exceptions to this.
+
+## 4. Overlay
+The `Presentation` module **MUST** have an `overlay` string property that returns one of the following values:
+
+| Value | Description |
+|-------|-------------|
+| `icon` | There is an informative icon, e.g. volume, on top of the app. |
+| `band`  | There is a horizontal overlay at the top or bottom of the app. |
+| `sidebar`     | There is a vertical sidebar covering less than 33% of the app on one side. |
+| `blocked` | There is a significantly sized UX covering a majority of the app. |
+
+## 5. Background Audio
+If an app has the `xrn:firebolt:capability:media:background-audio`, then it can keep playing audio/video when the app is in the `none` display state and the audio will be played for the user.
+
+When an app has this capability, it **MAY** be put into the `none` display state while in the `active` Lifecycle state.
+
+## 6. Picture-in-Picture Video
+If an app has the `xrn:firebolt:capability:media:picture-in-picture`, then it can keep playing audio/video when the app is in the `none` display state and the audio & video will be presented to the user in bounding box determined by the platform. Note that this does not include the entire UX of the app, just the active media pipeline.
+
+When an app has this capability, it **MAY** be put into the `none` display state while in the `active` Lifecycle state.
+
+## 7. Platform-provided Loading Screen
+Most apps will leverage a platform-provided loading screen.
+
+If an app provides the `xrn:firebolt:capability:presentation:loading-screen`
+capability, then the platform **MAY** use the app-provided loading screen, in
+which case, the rest of the section does not apply.
```

Comment:
I think the rest still applies in that case since it may take time to load the loading screen?